.github/workflows/Tests.yaml

```diff
@@ -253,11 +253,15 @@ jobs:
         run: |
           make doc-coverage
       # Upload as an artifact to make available to PRs
+      - name: Prepare for upload
+        working-directory: /work/build
+        run: |
+          zip -qr docs-html.zip docs/html
```

Comment:
`tar.gz`s are more common in Linux land, but I don't know if that really matters :)

Reply:
Probably doesn't matter, I'll leave as-is unless someone wants me to change this